### ISSUE TITLE
error raised in web version but not command line version

### ISSUE THREAD

**Issue by jbwexler** [bids-standard/legacy-validator#334]
I get this error when using the web version of bids-validator (including on openneuro). I don't get it when running the command line version, which is up-to-date with github.

Error: 1
Participant labels found in this dataset did not match the values in participant_id column found in the participants.tsv file.1 file
participants.tsv
0.984 KB | text/tab-separated-values
Location:

ds210/participants.tsv
Reason:

Participant labels found in this dataset did not match the values in participant_id column found in the participants.tsv file.
Evidence:

participants.tsv: 01, 02, 03, 04, 05, 06, 07, 08, 09, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31 folder structure: 01, 02, 03, 04, 05, 06, 07

**Comment by chrisgorgo** [comment]
Is the error correct? Do you have any empty sub- directories?

**Comment by suyashdb** [comment]
@chrisfilo i was also able to replicate the issue. Joe and myself both didnt get any errors in the developer tools either.

**Comment by chrisgorgo** [comment]
Let me rephrase - do you expect the error not to be raised in the web version of the validator (because it's incorrect) or to be raised in the command line (because indeed there is a discrepancy between participants.tsv and sub folders)?

Are there any empty sub folders?

**Comment by suyashdb** [comment]
sub folders are not empty. 

I expect not to generate the error in web versions. 
Normal behavior should be `no error` as participants id matches with subjects in the dataset directory. Command line tool shows normal behavior without generating any error.

**Comment by suyashdb** [comment]
@jbwexler @chrisfilo 

Ok that was a totally anomalous behavior. 
I deleted the dataset which was generating error, restarted my machine. Now i am not getting the error similar to command line validator tool. 
Both INCF validator.io and openneuro are not generating error which is expected. 

@jbwexler can you also download the data again from tacc and restart your machine, and confirm same behavior?

**Comment by jbwexler** [comment]
That fixed it for me too! Weird...
